pylxd/client.py

```diff
@@ -29,6 +29,17 @@
 
 requests_unixsocket.monkeypatch()
 
+LXD_PATH = '.config/lxc/'
+SNAP_ROOT = '~/snap/lxd/current/'
+APT_ROOT = '~/'
+if os.path.exists(os.path.expanduser(SNAP_ROOT)):
+    CERTS_PATH = "{}{}".format(SNAP_ROOT, LXD_PATH)
+else:
+    CERTS_PATH = "{}{}".format(APT_ROOT, LXD_PATH)
+DEFAULT_CERTS = (
+    os.path.expanduser('{}client.crt'.format(CERTS_PATH)),
+    os.path.expanduser('{}client.key'.format(CERTS_PATH)))
```

Comment:
And here too, please.  It will stop bizarre errors in the future when someone edits the path and leaves a separator off.